frontend/packages/console-shared/src/utils/application-utils.ts

```diff
@@ -0,0 +1,77 @@
+import { k8sList, k8sPatch, K8sResourceKind, modelFor } from '@console/internal/module/k8s';
+import * as _ from 'lodash';
+
+const updateItemAppLabel = (item: K8sResourceKind, application) => {
```

Comment:
Missing param and return types.
Shared code would be good to have a comment describing the purpose.